### PR TITLE
Made entire card clickable!

### PR DESCRIPTION
**Pull request by zacharyjarnagin**
Went into the for loop of objects in _curriculum/templates/curriculum/lessonplan_find_share.html_ and added surrounded the div in a span which I then surrounded in a link tag. Had to do some in-line HTML styling (which is not the best solution) to make sure the card styles did not change. 

I also formatted the HTML in PyCharm so that kinda sent things into a mess.

### CODE REVIEW

File: curriculum/templates/curriculum/lessonplan_find_share.html [hunk 1]
```diff
@@ -67,53 +72,63 @@ <h1 class="font-weight-bold find-share-curricula">Find & Share Curricula</h1>
...
+                                        <i class="fas fa-star" aria-hidden="true"></i>
+                                        <i class="fas fa-star" aria-hidden="true"></i>
+                                    </p>
+                                {% endif %}
+                                {% if lessonplan.web_only %}
+                                    <p class="pr-4" title="Chromebook compatible"><i
+                                            class="text-right fab fa-chrome"></i></p>
+                                {% endif %}
+                            </div>
+                            <h6 class="curriculum-text">{{ lessonplan.summary }}</h6>
```
Comment (jaedynlee) [inline curriculum/templates/curriculum/lessonplan_find_share.html]: This should still be a `p` tag, not `h6`

Comment (zacharyjarnagin) [reply curriculum/templates/curriculum/lessonplan_find_share.html]: Soo leaving it as a p tag turns the description font size to be much smaller than what it was initially before it was in the anchor tag. If you want me to change it back I can

Comment (jaedynlee) [reply curriculum/templates/curriculum/lessonplan_find_share.html]: Oh I see. The `curriculum a` class is overriding the entire card. I think we should keep the text as it was before, so I'd suggest either changing the `curriculum-text` class to specify `font-size=1rem` OR changing the `curriculum a` class to be something more specific like `curriculum-link` and using that only for the `View more details` tag.

Comment (zacharyjarnagin) [reply curriculum/templates/curriculum/lessonplan_find_share.html]: Okay I will try and implement that!

File: curriculum/templates/curriculum/lessonplan_find_share.html [hunk 2]
```diff
@@ -123,7 +123,7 @@ <h6 class="curriculum-title text-primary">{{ lessonplan.title }}</h6>
                                             class="text-right fab fa-chrome"></i></p>
                                 {% endif %}
                             </div>
-                            <h6 class="curriculum-text">{{ lessonplan.summary }}</h6>
+                            <p class="curriculum-text">{{ lessonplan.summary }}</p>
```
Comment (zacharyjarnagin) [inline curriculum/templates/curriculum/lessonplan_find_share.html]: Changed tags!

File: curriculum/templates/curriculum/lessonplan_find_share.html (hunk 1 above)
Comment (zacharyjarnagin) [reply curriculum/templates/curriculum/lessonplan_find_share.html]: Done! You should see that the curriculum text is a `p` element now and I added a CSS rule for `curriculum-text`. I noticed it was already a class being referenced, but couldn't find it in _main.css_. Let me know you're thoughts!

**Pull request merged**